src/app/tests/suites/certification/Test_TC_S_2_3.yaml

```diff
@@ -184,7 +180,7 @@ tests:
           "Step 2b: TH sends a AddScene command to DUT with the GroupID field
           set to G1, the SceneID field set to 0x01, the TransitionTime field set
           to 0x0001 (1s) and a set of extension fields appropriate to AC1."
-      PICS: S.S.C40.Rsp && PICS_SDK_CI_ONLY
+      PICS: (!S.S.C40.Rsp) && PICS_SDK_CI_ONLY
```

Comment:
This does not seem to match the test plan.

Reply:
Yes but the test plan would have the AddScene overwrite the EnhancedScene, so I assumed this would make sense to ignore. Not that it makes a difference in the expected outcome.

I can add the check on S.S.C00.R but isn't this step redundant if S.S.C40.Rsp is true?

Reply:
I think that if the test plan is wrong we should:

1. Raise a test plan issue.
2. Add a comment here pointing to that test plan issue.

I agree the test plan looks pretty weird to me.

Reply:
Issue [here](https://github.com/CHIP-Specifications/chip-test-plans/issues/3509), comment added